.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -1,23 +0,0 @@
----
-name: 💡 Feature request
-about: Suggest an idea for this project
-title: ''
-labels: community, feature-request
-assignees: ''
-
----
-
-**Is your feature request related to a problem? Please describe.**
-<!-- A clear and concise description of what the problem is. Ex. I'm always frustrated when [...] -->
-
-**Describe the goal of the feature**
-<!-- A clear and concise description of what you want to happen, and how it may used or behave. -->
-
-**Describe alternatives you've considered**
-<!-- Optionally, provide description of any alternative solutions or features you've considered. -->
-
-**Additional context**
-<!-- Add any other context or screenshots about the feature request here. -->
-
-**How does `datadog` help you?**
-<!-- Optionally, tell us why and how you're using datadog, and what your overall experience with it is! -->
```

Comment:
Hey, would it be reasonable to also re-add the "help you" question to the feature requests?